scripts/bump-version.sh

```diff
@@ -17,8 +17,8 @@ main() {
   new=$(_bump_version "$old" "$@")
   echo "Updating version from $old to $new"
 
-  sed -i "s|version = '$old'|version = '$new'|" build.gradle
-  sed -i "s|$old|$new|" README.md
+  sed -i '' "s|def projectVersion = '$old'|def projectVersion = '$new'|" build.gradle
+  sed -i '' "s|$old|$new|" README.md
```

Comment:
What is the `''` for?  I've seen this other places but IME it isn't reliably portable.

Reply:
Yeah, good catch. That was my change when working on mac...